src/assets/wise5/components/openResponse/edit-open-response-advanced/edit-open-response-advanced.component.ts

```diff
@@ -261,6 +273,10 @@ export class EditOpenResponseAdvancedComponent extends EditAdvancedComponentComp
     }
   }
 
+  protected toggleShowIdeaDescriptionsClicked(): void {
+    this.showIdeaDescriptions = !this.showIdeaDescriptions;
+  }
+
```

Comment:
Remove "Clicked" from the function name? 
```suggestion
  protected toggleShowIdeaDescriptions(): void {
    this.showIdeaDescriptions = !this.showIdeaDescriptions;
  }

```